ambry-router/src/test/java/com/github/ambry/router/RouterTestHelpers.java

```diff
@@ -312,7 +312,7 @@ static void compareContent(byte[] originalContent, ByteRange range, ReadableStre
   static List<ChunkInfo> buildChunkList(ClusterMap clusterMap, BlobId.BlobDataType blobDataType, long ttl,
       LongStream chunkSizeStream) {
     return chunkSizeStream.mapToObj(
-        chunkSize -> new ChunkInfo(getRandomBlobId(clusterMap, blobDataType), chunkSize, ttl))
+            chunkSize -> new ChunkInfo(getRandomBlobId(clusterMap, blobDataType), chunkSize, ttl))
```

Comment:
rm formatting edits. 
PR can be smaller if all formatting edits are moved to a different PR.